From ae58851b6d5e7348addec2d261e9c25d88ee339a Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Thu, 15 Feb 2024 16:29:09 +0100
Subject: [PATCH] Update generalinfo.f90

---
 .../RuNNer-v1_20---2020-09-17/src/generalinfo.f90            | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/release-versions/RuNNer-v1_20---2020-09-17/src/generalinfo.f90 b/release-versions/RuNNer-v1_20---2020-09-17/src/generalinfo.f90
index a0cdb7f..e80a3fe 100644
--- a/release-versions/RuNNer-v1_20---2020-09-17/src/generalinfo.f90
+++ b/release-versions/RuNNer-v1_20---2020-09-17/src/generalinfo.f90
@@ -57,13 +57,8 @@ subroutine generalinfo()
      !read(time,'(i2,i2,f6.3)')nhours,nminutes,seconds
      !read(date,'(i4,i2,i2)') year,month,day
      !directory=getcwd()
-#ifdef __INTEL_COMPILER
-      machname=hostnm()
-      directory=getcwd()
-#else
       ierr=hostnm(machname)
       ierr=getcwd(directory)
-#endif
 
       call getlog(username)
       call date_and_time(date,time)
