From e2f3415ad56095cec77e3e4c79b102fc9f297787 Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Thu, 15 Feb 2024 15:07:03 +0100
Subject: [PATCH] Update GNUmakefile

---
 src-devel/GNUmakefile | 28 ----------------------------
 1 file changed, 28 deletions(-)

diff --git a/src-devel/GNUmakefile b/src-devel/GNUmakefile
index b4491f8..7da4c2d 100644
--- a/src-devel/GNUmakefile
+++ b/src-devel/GNUmakefile
@@ -13,7 +13,6 @@
 
 # By default the static release version based on MKL will be made using ifort.
 VARS   = FC DEBUG MKL STATIC ARCH PFUNIT
-FC     = ifort
 DEBUG  = OFF
 MKL    = ON
 STATIC = ON
@@ -69,33 +68,6 @@ SRCS = $(shell $(FIND) "*$(SRCEXT)")
 # Object files will be stored in a build directory.
 OBJS = $(filter-out $(MAIN), $(addprefix $(BLDDIR), $(SRCS:$(SRCEXT)=$(OBJEXT))))
 
-# Compiler flags
-ifeq ($(FC),ifort)
-  FCFLAGS = -I$(MODDIR) -module $(MODDIR) -cpp $(OPT) -parallel -qopenmp \
-            -mtune=core-$(ARCH) -march=core-$(ARCH) -no-fast-transcendentals
-  LDFLAGS = $(FCFLAGS)
-  DBFLAGS = -check bounds -g -traceback
-else ifeq ($(FC),ifx)
-  FCFLAGS = -I$(MODDIR) -module $(MODDIR) -cpp $(OPT) -qopenmp \
-            -mtune=core-$(ARCH) -march=core-$(ARCH)
-  LDFLAGS = $(FCFLAGS)
-  DBFLAGS = -check bounds -g -traceback
-	STATIC  = OFF #does currently not work
-else ifeq ($(FC),gfortran)
-  ALLOWAM = $(shell $(FC) -fallow-argument-mismatch 2>&1 \
-            | grep -q 'unrecognized' || echo -fallow-argument-mismatch)
-  FCFLAGS = -I$(MODDIR) -J$(MODDIR) -cpp $(OPT) -m$(ARCH) -fopenmp -m64 $(ALLOWAM)
-  LDFLAGS = -O3 -m$(ARCH) -m64
-  DBFLAGS = -O3 -Wall -Wconversion -g -fbacktrace -fbounds-check
-else
-  $(error Unsupported compiler requested!)
-endif
-
-ifeq ($(DEBUG),ON)
-  FCFLAGS += $(DBFLAGS)
-  LDFLAGS += $(DBFLAGS)
-endif
-
 # Libraries
 #
 # Static linking is introduced by compile flags. It's a feature (some people
